src/youtube-tracker.coffee

```diff
@@ -14,9 +13,43 @@
 # Author:
 #   Karl Birch <mbwkarl@gmail.com>
 
+GOOGLE_API_KEY = process.env.GOOGLE_API_KEY
+
+querystring = require 'querystring'
+url = require 'url'
+
 module.exports = (robot) ->
-  robot.respond /hello/, (res) ->
-    res.reply "hello!"
+  if not GOOGLE_API_KEY?
+    return robot.logger.error "hubot-youtube-tracker: Missing GOOGLE_API_KEY in environment.  Please set and try again."
+
+  robot.respond /(?:tracker|ytt) recent (.*)/i, id: "tracker.recent", (res) ->
```

Comment:
Also, should we use `tracker.recent` as the id? I'm not sure if it's specific enough. Would `youtube.tracker.recent` be better?